src/inputtypeconversion.js

```diff
@@ -0,0 +1,10 @@
+export function convertFakePasswordInput () {
+  const collection = document.getElementsByClassName('fake_password_input')
+
+  for (let i = 0; i < collection.length; i++) {
+    const input = collection[i]
+    input.type = 'text'
+  }
+
+  console.log('Updated fake password input fields to text.')
```

Comment:
If you really want to print to the console, `console.debug` is probably better. Normally I don't print anything to the console though. In particular if the collection was empty, and no fake password input fields were actually updated.